### PR TITLE
Awrvi 2.0

### LINKED ISSUE #165: Update categories to AWRVI 2.0 research

### PR DESCRIPTION
**Pull request by smacfarlane**
closes #165 

@gina-alaska/awrvi Please review the awrvi-2.0 yml to ensure I haven't made any mistakes converting from 1.0.

I still need to make a number of changes for the interface to ensure 2.0 is the only version available through the UI.

### CODE REVIEW

File: db/seeds/categories/awrvi-2.0.yml [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+      - index: 0.5
+        description: "> 75% continous permafrost"
+      - index: 0.75
+        description: "1-75% continous permafrost and 1-75% no permafrost and 0% discontinous permafrost"
+      - index: 1.0
+        description: ">75% no permafrost"
+  - name: Physical - subsistence habitat
+    short_name: Subsistence habitat
+    categories:
+    - name: Anadromous Stream kilometers (%)
```
Comment (teknofire) [inline db/seeds/categories/awrvi-2.0.yml]: Is the (%) supposed to be here, the text and values seems to me to want to imply KM or even KM^2

Comment (smacfarlane) [inline db/seeds/categories/awrvi-2.0.yml]: Nope,  this is how it's defined in the datasheet I was given:
`% anadromous stream kilometers (from anadromous streams catalog)`

**Comment by grimm** [comment]
:+1:

**Comment by teknofire** [comment]
👍  

gonna add a question issue about the % vs KM stuff on that indicator

**Pull request merged**